__tests__/app.test.js

```diff
@@ -87,6 +87,20 @@ describe('GET /api/articles', () => {
         .expect(200)
         .then((response) => {
             expect(response.body.articles).toBeSortedBy('created_at', { descending: true });
+            expect(typeof response.body.articles).toBe('object');
```

Comment:
This is a good attempt and you're very close. We are dealing with many articles and will need to test all of them rather tha just the first one. How could we use iteration here to help us?